dipy/reconst/dti.py

```diff
@@ -1163,6 +1165,42 @@ def predict(self, gtab, S0=1):
         return tensor_prediction(self.model_params[..., 0:12], gtab, S0=S0)
 
 
+def iter_fit_tensor(step=1e4):
```

Comment:
Yes `step` is the number of voxels in one chunk. I will push a PR putting a params section, probably during the weekend.